ci-operator/jobs/cri-o/cri-o/cri-o-cri-o-main-presubmits.yaml

```diff
@@ -1,21 +1,5 @@
 presubmits:
   cri-o/cri-o:
-  - agent: jenkins
-    always_run: true
-    branches:
-    - ^main$
-    - ^main-
-    - ^release-1\.2$
-    - ^release-1\.2-
-    context: ci/kata-jenkins
```

Comment:
I think this one is running on the intel infra and we can keep it. right @mrunalp?

Reply:
fixed.

Reply:
yeah, let's keep this one. good catch!